spec/classes/collectd_plugin_modbus_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+
+describe 'collectd::plugin::modbus' do
+  on_supported_os(baseline_os_hash).each do |os, facts|
+    context "on #{os} " do
+      let :facts do
+        facts
+      end
+      let :pre_condition do
+        'include collectd'
+      end
+
+      options = os_specific_options(facts)
```

Comment:
the rspec variable `facts` and the local variable `facts` are always a bit confusing, so we try to distinguish a bit:
```suggestion
  on_supported_os(baseline_os_hash).each do |os, os_facts|
    context "on #{os} " do
      let :facts do
        os_facts
      end
      let :pre_condition do
        'include collectd'
      end

      options = os_specific_options(os_facts)
```